lib/log_bench.rb

```diff
@@ -15,9 +16,8 @@ class Error < StandardError; end
   extend LogBench::Parse
   extend LogBench::Setup
 
-  class << self
-    attr_accessor :configuration
-  end
+  singleton_class.attr_accessor :configuration
```

Comment:
I can revert back to the way it was before if you prefer, LMK. For me it was easier to see that we are setting a default config here immediately.